app/modules/intelligence/tracing/phoenix_tracer.py

```diff
@@ -0,0 +1,215 @@
+"""
+Phoenix Tracing Integration for LLM Monitoring
+
+This module sets up Arize Phoenix tracing for monitoring:
+- Pydantic AI agent operations (agent runs, delegations, structured outputs)
+- LLM API calls (via LiteLLM - Anthropic, OpenAI, etc.)
+- Token usage and costs
+- Multi-agent delegations (supervisor → subagents)
+- Tool calls and results
+- Performance metrics and latency
+
+CRITICAL SETUP ORDER:
+1. Call initialize_phoenix_tracing() at application startup (in main.py)
+2. This registers Phoenix and instruments Pydantic AI BEFORE any agents are created
+3. Create agents with instrument=True to enable tracing
+
+Usage:
+    from app.modules.intelligence.tracing.phoenix_tracer import initialize_phoenix_tracing
+
+    # Initialize once at application startup (BEFORE creating any agents)
+    initialize_phoenix_tracing()
+
+    # Then create agents with instrument=True
+    agent = Agent(model=..., tools=..., instrument=True)
+
+What gets traced:
+    - Pydantic AI: Agent.run(), Agent.run_sync(), agent.iter(), structured outputs, retries
+    - LiteLLM: completion(), acompletion(), streaming calls
+    - Multi-agent system: All supervisor and subagent interactions
+    - Tool calls: Function calls and results
+    - Tokens: Usage and cost tracking
+"""
+
+import os
+from typing import Optional
+from app.modules.utils.logger import setup_logger
+
+logger = setup_logger(__name__)
+
+# Global flag to track if Phoenix is initialized
+_phoenix_initialized = False
```

Comment:
Global variables should be in capital letters.
```suggestion
_PHOENIX_INITIALIZED = False
```